### ISSUE TITLE
BUG: Table sort order isn't indicated in the column header at all

### ISSUE THREAD

**Issue by rathboma** [beekeeper-studio/beekeeper-studio#1887]
Prior to 4.1, if you sorted a table by a specific column it would show a little up or down arrow to indicate the direction of the sort. This is missing from the table in 4.1.

**Comment by tomonetml** [comment]
plus ordering seems no more possible by clicking on the table header but just from the context menu

**Comment by rathboma** [comment]
Yes, that is correct, you have to right click or click the context menu to sort now.
